examples/shot_detect.py

```diff
@@ -215,7 +215,7 @@ def _verify_ffprobe():
         )
     try:
         out, err = proc.communicate()
-    except:
+    except Exception:
```

Comment:
This one I think we can leave as is, since its less clear what the child process will raise.  Googling suggests that typical types would be `OSError` and `ValueError`.

Reply:
And super minor, but I wonder if this exception block works even as is due to `out` and `err` being used before they are defined (but I might be missing something).

Reply:
Oops. Thanks for catching that. Fixed. Also, I've now actually tested that code path.